app/components/repo-actions.js

```diff
@@ -59,16 +59,24 @@ export default Component.extend({
     }
   }),
 
+  showPriorityBtn: computed('showPriority', function () {
+    if (this.showPriority === false) {
+      return false;
+    }
+
+    return true;
+  }),
+
   showPrioritizeBuildModal: false,
   canCancel: and('userHasPullPermissionForRepo', 'item.canCancel'),
   canRestart: and('userHasPullPermissionForRepo', 'item.canRestart'),
   canDebug: and('userHasPushPermissionForRepo', 'item.canDebug'),
   isHighPriority: or('item.priority', 'item.build.priority'),
   isNotAlreadyHighPriority: not('isHighPriority'),
   hasPrioritizePermission: or('item.permissions.prioritize', 'item.build.permissions.prioritize'),
-  canPrioritize: and('item.notStarted', 'isNotAlreadyHighPriority', 'hasPrioritizePermission'),
   insufficientPermissions: not('userHasPushPermissionForRepo'),
   tooltips: or('labelless', 'mobilelabels'),
+  canPrioritize: and('item.notStarted', 'isNotAlreadyHighPriority', 'hasPrioritizePermission', 'showPriorityBtn'),
```

Comment:
If you add a visibility property here, the property label becomes irrelevant. It doesn't indicate if user `canPrioritize` the build anymore, it does more. Please either update the property name or just combine two properties in template by `and()` helper - whatever you prefer.

Reply:
The reason why property names are important is that it may significantly confuse developers during debugging. Names must be explicit and descriptive.

Reply:
Got it. May you please verify now?